plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/CommitInfoEntry.java

```diff
@@ -25,6 +25,7 @@ public record CommitInfoEntry(
         String userName,
         String operation,
         Map<String, String> operationParameters,
+        Map<String, String> operationMetrics,
```

Comment:
Could you move this argument under `isBlindAppend`? That's the same order as Spark.

Reply:
Yeah ideally we expose those fields, so I can use them in the tests in vacuum operation PR. I will take care of that.